src/options.rs

```diff
@@ -92,7 +92,7 @@ impl Options {
     fn size_validator(s: String) -> Result<(), String> {
         match Options::parse_size(&s) {
             None => Err(format!("\"{}\" is not a valid size (in format \"NNNxMMM\")", s)),
-            Some((0, _)) | Some((_, 0)) => Err(format!("Can't resize image to size 0")),
+            Some((0, _)) | Some((_, 0)) => Err("Can't resize image to size 0".into()),
```

Comment:
Well, the [return type *must* be `Result<(), String>`](https://docs.rs/clap/2.14.0/clap/struct.Arg.html#method.validator), so that won't be a problem and `to_string()` expresses intent better in this case.